test/FastTests/Client/QueriesWithCustomFunctions.cs

```diff
@@ -3052,6 +3077,18 @@ private class IndexQueryResult
             public string Name { get; set; }
             public string Friend { get; set; }
         }
+
+        private class Document
+        {
+            public string Id { get; set; }
+            public Result[] Results { get; set; }
+        }
+
+        private class Result
+        {
+            [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+            public decimal? ResultValue { get; set; }                
```

Comment:
What happens if this is also doing a Rename of the json property name?

Reply:
It seems to be broken:

            [JsonProperty("Renamed", NullValueHandling = NullValueHandling.Ignore)]
            public decimal? ResultValue { get; set; }

Produces:

    FROM Documents as s WHERE id() IN ($p0) SELECT { Id : id(s), Results : s.Results.map(function(x){return {ResultValue:x.ResultValue};}) }

Reply:
Yes, that needs to be fixed